### DISCUSSION TITLE
How to stop a feed from running/processing after it has started?

### DISCUSSION THREAD

**Discussion by thijskaspers** [engram-design/FeedMe#1016] [Ideas]
When we delete a feedme task from the queue manager, the feed processing still continues in the background. We've noticed this because many new "Updating search indexes" tasks are still being spawned, and also new assets are still being uploaded in the background.

This can be annoying when you've noticed a bug during import, and the feed has hundreds of entries to process.
What is the correct way of stopping a feedme import after it has started?

**Comment by ben-callaway** [comment]
I've found this. Only way to truly stop it is kill the task in FeedMe, restart Supervisor (if using), PHP and sometimes the server!

**Comment by shaunmolloy** [comment]
Also came across this.

Restarting services may not always be an option for shared servers.

I deleted the product type and added it back as a workaround.

**Comment by andykearney** [comment]
This is a thing for us too. I've found that I can 'sometimes' stop a feed processing from the queue manager, but only by clicking the 'x' on the row for the Feed Me job in the queue list. Hitting release all jobs doesn't stop the feed running in the background and consequently search index jobs being spawned. 

Stoping the job on its individual row, then hitting release all jobs does seem to work. Sometimes.

**Comment by mark-chief** [comment]
Currently sitting here waiting for 40k entries to import incorrectly because I'm unable to stop the feedme import. Stopping the queue manager did not work for me.
